### ISSUE TITLE
Fira Code fonts (all) looks more blurry than Fira Mono

### ISSUE THREAD

**Issue by CapSel** [tonsky/FiraCode#985]
Fira Code is a beautiful font but its somehow more blurry than Fira Mono. Is it by design? How can I make it less blurry - like in Fira Mono?
I've prepared two screen shot for comparison:
https://capsel.org/tmp/screenshot03.png
https://capsel.org/tmp/screenshot04.png

**Comment by tonsky** [comment]
Try changing font size. 13px should work best

**Comment by CapSel** [comment]
Yes I know that it looks better when I change its size. The point of using 9px is that there are more lines of code on screen. Original font does not have this "problem".

**Comment by voltagex007** [comment]
Even I too have the same problem. It isn't crisp (font should be crisp and clear for better vision over the code while coding, and thats why coders search for a font like that) and increasing the size makes NeoVim a lot funnier and bad and uncomfortable. I use 11px, it is blurry bit.

**Comment by victorbombella** [comment]
I had the same problem, and the solution was this:

https://github.com/tonsky/FiraCode/issues/1213

**Comment by CapSel** [comment]
I used partial workaround - I enabled autohinter. Problem now is barely noticeable.
